environments/qa/hiera/common.json

```diff
@@ -16,28 +36,35 @@
         "127.0.0.2"
     ],
 
+    "grasshopper::setup::tenant_appdisplayname_escaped": "My+Timetable+-+Beta",
+
     /* Grasshopper repository */
     "app_ui_path": "/opt/grasshopper-ui",
     "app_install_method": "git",
     "app_install_config": {
         "source": "https://github.com/CUL-DigitalServices/grasshopper.git",
-        "revision": "master"
+        "revision": "ccf099550f74dded246c2cee9cb91d1cc30e78d8"
```

Comment:
Why are we using a specific commit here? I guess we should always use the latest master.